runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -1246,6 +1274,64 @@ class VM_JFRConstantPoolTypes {
 		gcConfiguration->heapAddressBits = J9JAVAVM_REFERENCE_SIZE(vm) * 8;
 	}
 
+	static uintptr_t processNativeLibrariesCallback(const char *libraryName, void *lowAddress, void *highAddress, void *userData)
+	{
+		size_t libraryPathLength = strlen(libraryName);
```

Comment:
This should move after the `for` loop - we may not need to compute it.
Also, a better name is `libraryNameLength` to clarify its relation to `libraryName`.